tests/pydantic/test_nullability.py

```diff
@@ -68,26 +68,22 @@ def test_empty_lists_are_ignored():
     assert RootModel(field_a="a", list_items=[]).dict() == {
         "field_a": "a",
     }
-    assert RootModel(
-        field_a="a", list_items=[ListItem(), ListItem(), ListItem()]
-    ).dict() == {"field_a": "a"}
+    with pytest.raises(pydantic.ValidationError):
```

Comment:
I'd also leave a variant with `ListItem()` to ensure we have a validation error there. We should change the name of this test.

Reply:
Agree. I created an extra issue(#42) to address this.